langchain_tools_demo/main.py

```diff
@@ -95,6 +93,20 @@ async def chat_handler(request: Request, prompt: str = Body(embed=True)):
         raise HTTPException(status_code=500, detail=f"Error invoking agent: {err}")
 
 
+@app.post("/reset")
+async def reset(request: Request):
+    """Reset agent"""
+    global user_agents
+    uuid = request.session["uuid"]
+    if uuid in user_agents.keys():
+        user_agent = user_agents[uuid]
+        await user_agent.client.close()
+        del user_agent.agent
+        del user_agents[request.session["uuid"]]
+    request.session.clear()
+    print(user_agents)
```

Comment:
Is this for debugging? Should it stay in the current implementation?

Reply:
I was thinking it would be helpful for the user to learn which agents are currently active. But I could remove this line if it is not necessary.